criu/proc_parse.c

```diff
@@ -1,3 +1,4 @@
+#include "pidfd.pb-c.h"
```

Comment:
Would it be possible to move this include further down, to be next to the other `*.pb-c.h`?

Reply:
Done!